neuralmonkey/trainers/generic_trainer.py

```diff
@@ -25,8 +26,9 @@
 class GenericTrainer(object):
 
     def __init__(self, objectives: List[Objective],
-                 l1_weight=0.0, l2_weight=0.0,
-                 clip_norm=False, optimizer=None, global_step=None) -> None:
+                 l1_weight: float=0.0, l2_weight: float=0.0,
+                 clip_norm: Optional[float]=None, optimizer=None,
```

Comment:
to tam bylo clip norm je bool a prošlo to jo? :-) to proto že to je všude nastavený na 1.0?